### ISSUE TITLE
Missing functionalities around Playlists

### ISSUE THREAD

**Issue by alex-nexus** [alejandrobabio/music-api#5]
```
Playlists
creating and deleting playlists
changing playlist name
adding/removing songs from the playlist
```

**Comment by alejandrobabio** [comment]
Yes, there are many missing features: the update is not done, nor the queries (which were not on the list but will be needed). Nor the upload of images, much less streaming. But it is still a WIP. As for adding/removing songs from the playlist, it will work in the same way as for add/remove songs from Album (which it is programmed).
